### DISCUSSION TITLE
How to achieve only one online user for one account at a time

### DISCUSSION THREAD

**Discussion by xibei312** [fastapi-users/fastapi-users#732] [Q&A]
The first time I use Fastapi-Users, I want to know how to use the JWT authentication mode of Fastapi-Users to realize that only one user can be online at the same time

**Comment by frankie567** [comment]
Hi @xibei312!

I don't really understand what you want to achieve 🤔 In a typical web application, users create *sessions* (in the form of a JWT, a cookie or whatever access token) granting them access to the service and their data. There is no concept of *one user at a time*.

To achieve this, you would need some sort of state on the backend side, like a database or a Redis key store to store the *single current user* and add logic to kick out other ones.

I don't really see the use case behind that, but you may achieve this by implementing [your own authentication backend](https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/authentication/base.py).

**Reply by xibei312** [reply]
I mean is that one account cannot be logged in to multiple devices at the same time, I don't know how to do that
